docs/storage/modules/ROOT/pages/aws-s3.adoc

```diff
@@ -121,3 +124,51 @@ Clojure::
                        ...}}
 ----
 ====
+
+
+=== AWS Transfer Manager
+Also AWS Transfer Manager works fine with the regular `S3AsyncClient` it is recommended to use the new
+link:https://aws.amazon.com/blogs/developer/introducing-crt-based-s3-client-and-the-s3-transfer-manager-in-the-aws-sdk-for-java-2-x/[CRT-based S3 client] in order to gain its full benefit.
+The following snippet
+
+[tabs]
+====
+Clojure::
++
+[source,clojure]
+----
+(defn- s3-configurator [_]
```

Comment:
can we give users a utility for this? maybe (see below comment) if we make a different module for the xfer-mgr we can give it a different default for S3 configurator too

Reply:
Ok --  do you reckon we should use the CRT S3 Async client by default? That would be the only sensible configuration. 
Not sure it's the right thing to do. The doc gives an example of how to use `S3Configurator` to configure a CRT Client with options.
I'd think someone having to deal with this would refer to the AWS CRT S3 Async Client documentation. No?